lib/tasks/default.rake

```diff
@@ -3,5 +3,5 @@ if Rails.env.test? || Rails.env.development?
   RuboCop::RakeTask.new
 
   task(:default).clear
-  task default: [:rubocop, :test, "test:system"]
+  task default: [:rubocop, :test, "test:system", :spec]
```

Comment:
not sure why, but looks like this is not working in the CI, i think it is not running `rspec` for some reason

Reply:
great catch! I figured it out. We needed to add the `rspec` gem in the `:development, :test` group instead of just `:development`